tests/unit/ui/components/TestButton.js

```diff
@@ -34,7 +34,7 @@ describe('<Button />', () => {
   it('renders a link with a href', () => {
     const href = 'https://addons.mozilla.org';
     const button = shallow(
-      <Button className="Bar" href={href}>Link text!</Button>);
+      <Button className="Bar" href={href} type="none">Link text!</Button>);
```

Comment:
ah, `none` means `link` like... `<a>`?

Reply:
I'm late to this response, but yes. It's there for a few things we want to use mostly unstyled buttons for.